webapp/pages/index.vue

```diff
@@ -119,34 +120,34 @@ export default {
               })
             }
           }
-        }).then(({ data }) => data && data.deletePost)
+        }).then(({data}) => data && data.deletePost)
       } catch (e) {
         console.log(e)
       }
     },
-    async upvotePost (id) {
+    async upvotePost(id) {
       try {
         await this.$apollo.mutate({
           mutation: upvotePost,
-          variables: { id },
-        }).then(({ data }) => data && data.upvotePost)
+          variables: {id},
+        }).then(({data}) => data && data.upvotePost)
       } catch (e) {
         console.log(e)
       }
     },
-    async downvotePost (id) {
+    async downvotePost(id) {
       try {
         await this.$apollo.mutate({
           mutation: downvotePost,
-          variables: { id },
-        }).then(({ data }) => data && data.downvotePost)
+          variables: {id},
+        }).then(({data}) => data && data.downvotePost)
       } catch (e) {
         console.log(e)
       }
     }
   },
   computed: {
-    sortedNewsListItems () {
+    sortedNewsListItems() {
```

Comment:
Did you change your lint configurations? :thinking:

Having these changes creates noise and distracts from relevant changes. You probably want to exclude those changes from the PR being reviewed.